portfolio/src/main/webapp/index.html

```diff
@@ -28,6 +28,7 @@
         </div>
         </div>
       </nav>
+<<<<<<< HEAD
```

Comment:
The changes in this commit add merge conflict artifacts back into the code. Revert this commit and the PR looks good.